pytorch_lightning/profiler/profilers.py

```diff
@@ -504,7 +535,7 @@ def summary(self) -> str:
         output_string = f"{os.linesep}Profiler Report{os.linesep}"
         for action, stats in recorded_stats.items():
             output_string += (
-                f"{os.linesep}Profile stats for: {action}{os.linesep}{stats}"
+                f"{os.linesep}Profile stats for: {action} rank: {self.local_rank} {os.linesep}{stats}"
```

Comment:
In the profiler connector:
```     python   
local_rank = trainer.local_rank if trainer.world_size > 1 else None
```

So does that mean we can print?

```
.... rank: None ....
```

I think we should just default to rank 0 for the local rank when profiling, or omit the rank from the output string

Reply:
Done !